### PR TITLE
Add tool_deps tool to allow triggering installation of local tool dep…

### PR DESCRIPTION
**Pull request by innovate-invent**
…endencies

The contribution docs don't detail how to do tests. This tool requires that a tool has already been manually installed into galaxy.

It also isn't clear how to provide a command line wrapper to execute the python script

### CODE REVIEW

File: src/ephemeris/tool_deps.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+from ephemeris import get_galaxy_connection
+from ephemeris.common_parser import get_common_args
+
+
+def _parser():
+    parent = get_common_args()
+    parser = argparse.ArgumentParser(parents=[parent])
+    parser.add_argument("-t", "--tool", help='Path to a tool file or tool_conf file')
+    parser.add_argument("-i", "--id", help='Comma seperated list of tool ids')
```
Comment (bgruening) [inline src/ephemeris/tool_deps.py]: You could use nargs here.

File: docs/commands.rst [hunk 2]
```diff
@@ -10,5 +10,6 @@ Commands
    commands/run-data-managers
    commands/setup-data-libraries
    commands/shed-tools
+   commands/tool-deps
```
Comment (bgruening) [inline docs/commands.rst]: Maybe install-tool-deps?

Comment (bgruening) [reply docs/commands.rst]: If you could add sub-commands like show, install, remove, reinstall ... then this name would make sense imho. Do you plan to add those?

Comment (innovate-invent) [reply docs/commands.rst]: I was thinking of having sub-commands but wanted feedback before implementing all of that :)

I'll take a crack at them tomorrow.

Comment (innovate-invent) [reply docs/commands.rst]: Looks like bioblend provides no means of inspecting or uninstalling a tool dependency

**Comment by mvdbeek** [comment]
This requires either the tool xml file or a tool_conf.xml, neither of which are formats we deal with in ephemeris and that are very local to Galaxy. I'd suggest a simple yaml list of tool ids.

**Comment by innovate-invent** [comment]
@mvdbeek It does take a list of tool ids on the command line. The tool_conf.xml support is useful when deploying a container and ephemeris is connecting to localhost with direct access to galaxies filesystem.

**Comment by mvdbeek** [comment]
I understand the utility, but in principle that's not the scope of ephemeris. Still this mostly works via the API, so if you can add loading a plain list of tool ids via `yaml.safe_load` and make that the default for `-t` I'm OK to merge this.

File: docs/commands/install-tool-deps.rst [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Workflow-install
```
Comment (mvdbeek) [inline docs/commands/install-tool-deps.rst]:
```suggestion
Install-tool-deps
```

Comment (innovate-invent) [reply docs/commands/install-tool-deps.rst]: Thanks for catching that

**Comment by mvdbeek** [comment]
There are some linting errors now (https://travis-ci.org/galaxyproject/ephemeris/jobs/543810023#L475), the other failures should have been temporary and unrelated.

**Comment by innovate-invent** [comment]
The current lint failure is not relevant to this project. The security implication is that it could fill RAM and crash the system if a malicious xml file is provided.

**Pull request merged**